olm/olmconfig.yaml

```diff
@@ -0,0 +1,57 @@
+# This configuration is a placeholder. Replace any values with relevant values for your
+# service controller project.
+---
+annotations:
+  capabilityLevel: Basic Install
+  shortDescription: AWS API Gateway v2 controller is a service controller for managing API Gateway v2 resources
+    in Kubernetes
+displayName: AWS Controllers for Kubernetes - Amazon API Gateway v2
+description: |-
+  Manage Amazon API Gateway v2 resources in AWS from within your Kubernetes cluster.
+  
+
+  **About Amazon API Gateway v2**
+
+
+  Amazon API Gateway is a fully managed service that makes it easy for developers to create, publish, maintain, monitor, and secure APIs at any scale. APIs act as the "front door" for applications to access data, business logic, or functionality from your backend services. Using API Gateway, you can create RESTful APIs and WebSocket APIs that enable real-time two-way communication applications. API Gateway supports containerized and serverless workloads, as well as web applications.
+
+
+  API Gateway handles all the tasks involved in accepting and processing up to hundreds of thousands of concurrent API calls, including traffic management, CORS support, authorization and access control, throttling, monitoring, and API version management. API Gateway has no minimum fees or startup costs. You pay for the API calls you receive and the amount of data transferred out and, with the API Gateway tiered pricing model, you can reduce your cost as your API usage scales.
+
+  
+  **About the AWS Controllers for Kubernetes**
+
+  
+  This controller is a component of the [AWS Controller for Kubernetes](https://github.com/aws/aws-controllers-k8s)
+  project. This project is currently in **developer preview**. 
+samples:
+- kind: APIMapping
+  spec: '{}'
+- kind: API
+  spec: '{}'
+- kind: Authorizer
+  spec: '{}'
+- kind: Deployment
+  spec: '{}'
+- kind: DomainName
+  spec: '{}'
+- kind: IntegrationResponse
+  spec: '{}'
+- kind: Integration
+  spec: '{}'
+- kind: Model
+  spec: '{}'
+- kind: RouteResponse
+  spec: '{}'
+- kind: Route
+  spec: '{}'
+- kind: Stage
+  spec: '{}'
+- kind: VPCLink
+  spec: '{}'
+maintainers:
+- name: "Your Team Name"
+  email: "your-team@example.com"
```

Comment:
not sure if we discussed this on a previous repo, but did we want to set this to something like ack-infra@amazon.com?

Reply:
Ah, if we did - then I missed it. Each OLM config has a placeholder value in case each service controller maintainer's contact information is unique. I mentioned it in the notes, and fully expected those to change before we ever started adding these steps into any build pipelines.

Reply:
Not a problem at all. I'll let each service team update to their liking.